src/services/api/DatagouvfrAPI.js

```diff
@@ -1,10 +1,10 @@
 import axios from "axios"
 import config from "@/config"
-import { toast } from "vue3-toastify"
-import { useLoading } from "vue-loading-overlay"
+// import { toast } from "vue3-toastify"
```

Comment:
Remove unused code if this is definitive.

Also, you might want to at least log the error response below since it's not toasted anymore and this code is used everywhere. Logging might not be enough because I guess it would break the logic somewhere else.

Plus, the loading indicator will disappear almost everywhere, this might not be what you want.

Reply:
Pour cette PR au moins, on peut effectivement laisser le loader, car seulement le toast est bloquant.

@abulte l'idée est de renvoyer la réponse d'erreur pour qu'elle soit gérée au niveau du store et les composants.

Reply:
Yes (et bonne idée) mais en attendant elle ne l'est pas dans beaucoup de cas. Peut-être utiliser [`request` directement](https://github.com/ecolabdata/ecospheres-front/blob/91ceaedb9458bcf5b77b261e458a0f01a0ff8162/src/services/api/DatagouvfrAPI.js#L47) en attendant un refacto complet.

Reply:
On ne peut pas attendre, justement. Gérer cette réponse d'erreur fait partie du DoD de #60. Et c'est de toute manière de la dette technique, je préfère qu'on casse tout maintenant que plus tard.

Reply:
Je vais remettre pour le moment toastify